.codecov.yml

```diff
@@ -119,6 +123,14 @@ coverage:
         target: 75
         flags:
         - puma
+      PureFA:
+        target: 75
+        flags:
+        - purefa
+      PureFB:
+        target: 75
+        flags:
+        - purefb
```

Comment:
Not sure why this is changed, but if it's something you didn't manually change, can you keep the order that is currently in Master? You can probably just copy and paste the file from master and then add in your changes (gRPC entry and the gitea spacing fix). That should prevent the chance of a unwanted change being introduced.

Reply:
Copied `.codecov.yml` from https://github.com/DataDog/integrations-extras/blob/eb27eec10e422c0a7df4a9288881e92249766adb/.codecov.yml and added gRPC check to it. https://github.com/DataDog/integrations-extras/pull/1381/commits/5a0559be2afa6f1a72b73c590f35685490bc6130